test/integration/src/test/java/io/pravega/test/integration/KeyValueTableTest.java

```diff
@@ -192,6 +261,1427 @@ public void testCreateDeleteKeyValueTable() {
 
     }
 
+    /**
+     * Test case-42: Insert Same Key into multiple KeyFamilies
+     */
+    @Test
+    public void testSameKeyMutipleKF() {
+        final KeyValueTableInfo newKVT = newKeyValueTableName();
+        KeyValueTable<Integer, String> newKeyValueTable = getKeyValueTable(newKVT);
+
+        String kf1 = "KeyFamily1";
+        String kf2 = "KeyFamily2";
+
+        Integer key = 22;
+        String value1 = "This value belongs to KeyFamily1";
+        String value2 = "This value belongs to KeyFamily2";
+
+        Version version1 = newKeyValueTable.put(kf1, key, value1).join();
+        Version version2 = newKeyValueTable.put(kf2, key, value2).join();
+        log.info("Version1: {}", version1);
+        log.info("Version2: {}", version2);
+
+        TableEntry<Integer, String> tableEntry1 = newKeyValueTable.get(kf1, key).join();
+        TableEntry<Integer, String> tableEntry2 = newKeyValueTable.get(kf2, key).join();
+
+        assertEquals("Keys should be same for both entries",
+                tableEntry1.getKey().getKey(), tableEntry2.getKey().getKey());
+        assertEquals("Corresponding value for entry1 should be as inserted",
+                value1, tableEntry1.getValue());
+        assertEquals("Corresponding value for entry2 should be as inserted",
+                value2, tableEntry2.getValue());
+        assertNotEquals("Corresponding values should be different for both the entries",
+                tableEntry1.getValue(), tableEntry2.getValue());
+
+        log.info("Table entry1, key: {} and value: {}", tableEntry1.getKey(), tableEntry1.getValue());
+        log.info("Table entry2, key: {} and value: {}", tableEntry2.getKey(), tableEntry2.getValue());
+        log.info("Successfully completed test Insert Same Key into multiple KeyFamilies");
+    }
+
+    /**
+     * Test case-43: Conditional insert KVP, if key not exists already
+     */
+    @Test
+    public void testInsertIfExists() {
```

Comment:
This is already covered in the base class.

Reply:
Sure, I'll remove this test as its a duplicate.